babel/util.py

```diff
@@ -217,6 +218,12 @@ def wraptext(text: str, width: int = 70, initial_indent: str = '', subsequent_in
     :param subsequent_indent: string that will be prepended to all lines save
                               the first of wrapped output
     """
+    warnings.warn(
+        "babel.util.wraptext is deprecated and will be removed in a future version of Babel. "
```

Comment:
To be consistent with the rest of the text:
```suggestion
        "`babel.util.wraptext` is deprecated and will be removed in a future version of Babel. "
```